pkg/southbound/synchronizer/factory.go

```diff
@@ -219,8 +232,6 @@ func (cm *connectionMonitor) synchronize() error {
 		cm.operationalStateCacheLock.Lock()
 		delete(cm.operationalStateCache, cm.device.ID)
 		cm.operationalStateCacheLock.Unlock()
-		cm.southboundErrorChan <- events.NewErrorEventNoChangeID(events.EventTypeErrorDeviceConnect,
```

Comment:
good riddance

Reply:
Hmm... actually I realized this code was correct. The reason is because the factory loop maintains the `connections` map, and a synchronizer is only recreated if it was already connected before receiving a disconnect event. But I think we probably have to keep the `connections` map to avoid reconnecting multiple times when concurrent disconnection events occur.